### ISSUE TITLE
The AMP4Email documentation for the amp-list component is written incorrectly(for the src attribute)

### ISSUE THREAD

**Issue by alexandr-kazakov** [ampproject/amp.dev#4715]
### Problem
The AMP4Email documentation for the amp-list component is written incorrectly(for the src attribute)

![chrome_IMfezZMYxz](https://user-images.githubusercontent.com/20513262/94325536-90ab7b00-ffa7-11ea-9e0a-921e8fb6fb00.png)

The documentation for the `src `attribute of the second and third paragraph are incorrect, they belong to the AMP-pages. More precisely: the third point is definitely incorrect, amp-script does not work in AMP4Email. As for the second paragraph, I'm not sure, I was confused by it myself. I tried in AMP4Email to initiate amp-list from local amp-state and validator(https://amp.gmail.dev/) gives me an error, screenshot: 

![chrome_db5Lwa0tgk](https://user-images.githubusercontent.com/20513262/94325570-b6388480-ffa7-11ea-978f-1e946de3efb9.png)
 so I think the third paragraph also doesn't apply to AMP4Email.

### Location
https://amp.dev/documentation/components/amp-list/?format=email#src-(required)

### Information architecture 
We need to delete the second and third paragraphs in the AMP4Email documentation for the amp-list component, src attribute, as soon as possible. This is very confusing.

**Comment by manisaiprasad** [comment]
Hey @matthiasrohmer I would like to take on this issue

can I change this

> The URL of the remote endpoint that returns the JSON that will be rendered within this <amp-list>. 
>There are three valid protocols for the src attribute.
>1. https: This must refer to a CORS HTTP service. Insecure HTTP is not supported.
>2. amp-state: For initializing from ```<amp-state>``` data. See Initialization from ```<amp-state>``` for more details.
>3. amp-script: For using ```<amp-script>``` functions as the data source. See Using ```<amp-script>``` as a data source for more details.

To

The URL of the remote endpoint that returns the JSON that will be rendered within this ```<amp-list>```.  This must refer to a CORS HTTP service and Insecure HTTP is not supported.


Open to contribute😇

**Comment by CrystalOnScript** [comment]
Thanks for wanting to take this one on @manisaiprasad! I think that rewrite is nearly perfect, just a nit to change from passive to active voice:

```
The URL of the remote endpoint that returns the JSON for amp-list to render. This must refer to a CORS HTTP service and Insecure HTTP is not supported. 
```
The two text blocks need to be wrapped in format filtering blocks. How to do so is [outlined here](https://amp.dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/?format=websites#filtering-sections). The original section would need to filter for websites and stories, the updated section would be email only. 

Lastly, the documentation needs to be updated in the [amphtml](https://github.com/ampproject/amphtml/blob/master/extensions/amp-list/amp-list.md) repository. 

Let me know if you need any help or have any questions!

**Comment by alexandr-kazakov** [comment]
@CrystalOnScript Hi. Any progress here? It seems that this error is still present in the documentation.

**Comment by patrickkettner** [comment]
fixed in ampproject/amphtml#31588
